src/lib/integration_test_cloud_engine/graphql_polling_log_engine.ml

```diff
@@ -85,9 +85,9 @@ let rec filtered_log_entries_poll node ~logger ~event_writer
             [ ("node", `String (Node.infra_id node))
             ; ("err", Error_json.error_to_yojson err)
             ] ;
-        (* Declare the node to be offline. *)
+        (* Emit an event that declares the node to be down. *)
         Pipe.write_without_pushback_if_open event_writer
-          (node, Event (Node_offline, ())) ;
+          (node, Event (Node_down, ())) ;
```

Comment:
Why this renaming?

Reply:
This is not merely a renaming.  As I stated in the pr's description, I've split `node_offline` into `node_down` and `node_stopped`.  The difference is that node_stopped indicates that the user has started a deliberate shutdown, while `node_down` is automatically detected by the log engine which indicates the node being down for any reason, deliberate or otherwise.  This distinction is critical for determining if something wrong happened or not.